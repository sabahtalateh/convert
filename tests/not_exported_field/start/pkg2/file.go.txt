package pkg2

import "github.com/sabahtalateh/convert/tests/not_exported_field/work/pkg1"

type Out struct {
	a int32
}

//go:generate convert
func Convert(in *pkg1.In) *Out {}
