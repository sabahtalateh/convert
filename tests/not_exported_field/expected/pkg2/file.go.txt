package pkg2

import "github.com/sabahtalateh/convert/tests/not_exported_field/work/pkg1"

type Out struct {
	a int32
}

func Convert(in *pkg1.In) *Out {
	var out *Out


	return out
}
